graphql/quote.graphql

```diff
@@ -59,6 +59,10 @@ type Quote {
   viewedBySales: Boolean
   viewedByCustomer: Boolean
   salesChannel: String
+  seller: String
+  approvedBySeller: Boolean
+  parentQuote: String
```

Comment:
After the split, all quotes are independent. So there is no need to keep a parent quote.

Reply:
Without a link between the sellers quotes and the "parent" quote, it would not be possible to create this screen, for example: https://www.figma.com/design/SDETDCU5obc4Kg4WtmG6FH/RedCloud-%C2%B7-Seller-Quotes-%5BCubos%5D?node-id=263-24328&t=daYlrwPapSKRVfZN-4.

Reply:
The where assembly that brings the quotes at marketplace side was changed because of this, always adding `parentQuote is null` in searches: https://github.com/vtex-apps/b2b-quotes-graphql/pull/59/files#diff-9766252b6f3131ecfc785f3869d3129273b19c4a91b52963b62c5d3803c964d7

Reply:
The "link" between the quotes should be just a name or label and not a parent quote. It's name displayed in this [screen](https://www.figma.com/design/SDETDCU5obc4Kg4WtmG6FH/RedCloud-%C2%B7-Seller-Quotes-%5BCubos%5D?node-id=263-11890&t=7Ra2yl4bgWaIFGQ6-0)